### PR TITLE
RHDEVDOCS-4493: Create release note for Doc enterprise-4.12: Pipelines Usability & Customer Happiness

### PR DESCRIPTION
**Pull request by smunje1**
[RHDEVDOCS-4493](https://issues.redhat.com//browse/RHDEVDOCS-4493): Doc: Pipelines Usability & Customer Happiness

Aligned team: Dev Tools
OCP version for cherry-picking: enterprise-4.12
JIRA issues: [RHDEVDOCS-4493](https://issues.redhat.com//browse/RHDEVDOCS-4493)
Preview pages: [Download to see the preview in your browser](https://drive.google.com/file/d/1cGWZw0WaQ3UoUEpGi3y4KSxDJwjzGorF/view?usp=share_link)
SME Review: @vikram-raj 
QE review: @sanketpathak  
Peer-review: @sounix000 and @DebarghoGhosh

### CODE REVIEW

**Comment by smunje1** [comment]
Hello @vikram-raj 
Could you help with the SME review of this PR?

Thank you!

**Comment by smunje1** [comment]
A screenshot:
image

**Review by vikram-raj — APPROVED** [review]
LGTM

**Comment by smunje1** [comment]
Hello @sanketpathak 
Could you help with the QE review of this PR?

Thank you!

**Comment by smunje1** [comment]
After rebasing and resolving the merge conflicts, a screenshot:
image

**Comment by smunje1** [comment]
Hello @sanketpathak
Please help with the QE review of this PR.

Thank you!

**Comment by sanketpathak** [comment]
LGTM

File: release_notes/ocp-4-12-release-notes.adoc
Comment (DebarghoGhosh) [inline release_notes/ocp-4-12-release-notes.adoc:119]: Pipeline details repeated twice.

File: release_notes/ocp-4-12-release-notes.adoc
Comment (DebarghoGhosh) [inline release_notes/ocp-4-12-release-notes.adoc:121]: Hove over the tasks to see the task details.

File: release_notes/ocp-4-12-release-notes.adoc
Comment (DebarghoGhosh) [inline release_notes/ocp-4-12-release-notes.adoc:122]: Use the standard icons to zoom in, zoom out, fit to screen and reset the view.

File: release_notes/ocp-4-12-release-notes.adoc
Comment (DebarghoGhosh) [inline release_notes/ocp-4-12-release-notes.adoc:123]: Using "Additionally" is not required. It can be: You can hover over the tasks badge to see the total number of tasks and the completed tasks.

File: release_notes/ocp-4-12-release-notes.adoc
Comment (sounix000) [reply release_notes/ocp-4-12-release-notes.adoc:121]: "... page visualization performing the following actions ..." - not clear; something seems to be missing. Depending on the context, it can be "that performs" or "by performing". I don't know. You are the best judge of the context @smunje1 .

File: release_notes/ocp-4-12-release-notes.adoc
Comment (sounix000) [reply release_notes/ocp-4-12-release-notes.adoc:122]: "... fit to screen, and reset the view." 

Serial comma!

File: release_notes/ocp-4-12-release-notes.adoc
Comment (sounix000) [reply release_notes/ocp-4-12-release-notes.adoc:123]: +1 for Debargho's comments.

File: release_notes/ocp-4-12-release-notes.adoc
Comment (smunje1) [reply release_notes/ocp-4-12-release-notes.adoc:119]: @DebarghoGhosh The first instance is for **Pipeline** details and the second instance is for **PipelineRun** details page. So, we don't have repetition here.

File: release_notes/ocp-4-12-release-notes.adoc
Comment (smunje1) [reply release_notes/ocp-4-12-release-notes.adoc:122]: Based on comments for Line 122, I will rephrase it as follows:
"Use the standard icons to zoom in, zoom out, fit to screen, and reset the view."

**Review by sounix000 — APPROVED** [review]

**Comment by smunje1** [comment]
@sounix000 and @DebarghoGhosh Could you look at the latest changes and give "LGTM" if there are no more comments?
A screenshot:
image

**Comment by sounix000** [comment]
LGTM

**Comment by DebarghoGhosh** [comment]
LGTM

**Pull request merged**
